docs/EXTERNAL_CHECK_CREATION.md

```diff
@@ -90,7 +90,9 @@ Clients outside of Go can be found in the [clients directory](../clients).
 The following environment variables are injected into every checker pod that Kuberhealthy runs.  When writing your checker code, you can depend on these environment variables always being available to you, even if you do not specify them in your `khcheck` spec.
 ```
 KH_REPORTING_URL: The Kuberhealthy URL to send POST requests to for check statuses.
-KH_CHECK_RUN_DEADLINE: The Kuberhealthy-calculated deadline for checks given in unix.
+KH_CHECK_RUN_DEADLINE: The Kuberhealthy-calculated deadline for checks given in Unix.
```

Comment:
Maybe add that this dependent on kuberhealthy check timeout? and if timeout isn't set there's a default...

Reply:
done